lib/ui/scripts/config.js

```diff
@@ -0,0 +1,79 @@
+import path from 'path';
+import webpack from 'webpack';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import UglifyJsPlugin from 'uglifyjs-webpack-plugin';
```

Comment:
I've used `terser-webpack-plugin` in my previous PR, and looks like webpack uses it by default now.